src/main/java/com/nike/cerberus/auth/connector/onelogin/OneLoginAuthConnector.java

```diff
@@ -184,20 +184,21 @@ protected SessionLoginTokenData createSessionLoginToken(final String username, f
 
         final CreateSessionLoginTokenResponse createSessionLoginTokenResponse = oneLoginClient.createSessionLoginToken(username, password);
 
+        long statusCode = createSessionLoginTokenResponse.getStatus().getCode();
+
         if (createSessionLoginTokenResponse.getStatus().isError()) {
             String msg = String.format("The user %s failed to authenticate for reason: %s", username, createSessionLoginTokenResponse.getStatus().getMessage());
-            if (createSessionLoginTokenResponse.getStatus().getCode() == 400L) {
-                if (StringUtils.startsWithIgnoreCase(createSessionLoginTokenResponse.getStatus().getMessage(), "MFA")) {
-                    throw ApiException.newBuilder()
-                            .withApiErrors(DefaultApiError.MFA_SETUP_REQUIRED)
-                            .withExceptionMessage(msg)
-                            .build();
-                } else {
-                    throw ApiException.newBuilder()
-                            .withApiErrors(DefaultApiError.AUTH_BAD_CREDENTIALS)
-                            .withExceptionMessage(msg)
-                            .build();
-                }
+            if (statusCode == 400 &&
+                    StringUtils.startsWithIgnoreCase(createSessionLoginTokenResponse.getStatus().getMessage(), "MFA")) {
+                throw ApiException.newBuilder()
+                        .withApiErrors(DefaultApiError.MFA_SETUP_REQUIRED)
+                        .withExceptionMessage(msg)
+                        .build();
+            } else if (statusCode == 401) {
+                throw ApiException.newBuilder()
+                        .withApiErrors(DefaultApiError.AUTH_BAD_CREDENTIALS)
+                        .withExceptionMessage(msg)
+                        .build();
             } else {
                 throw ApiException.newBuilder()
                         .withApiErrors(DefaultApiError.GENERIC_BAD_REQUEST)
```

Comment:
This is also where that `The request will not be completed` message is getting generated from and needs to be cleaned up with a better message and a `401`